salt/utils/schedule.py

```diff
@@ -214,7 +214,7 @@ def _check_max_running(self, func, data, opts, now):
         # dict we treat it like it was there and is True
 
         # Check if we're able to run
-        if not data['run']:
+        if 'run' not in data or not data['run']:
```

Comment:
`_check_max_running` is called in 2 places (`eval` and `run_job`). `eval` guarantees that `data['run']` is set, now `run_job` does as well.

With current code, `'run' not in data` will always be False. I'm ok with defensive programming, just thought I'd point that out.

Reply:
If we change the PR for the other issue, maybe `not data.get('run', False)` would be better? Just emphasizing the important bit is 'run' being False, with a "just in case" second return.

Reply:
Don't worry about that last bit, I was overthinking it.

Reply:
Yeah. That makes more sense.  My fix was just a quick fix to get the functionality working again but this is a cleaner approach.